introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-incremental-training-highlevel.ipynb

```diff
@@ -10,9 +10,15 @@
     "  1. [Permissions and environment variables](#Permissions-and-environment-variables)\n",
```

Comment:
<p>Is this formatted correctly? Looks like there's a line break after "the" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
I don't see it on my side. Am I looking at the right paragraph? 

![Screenshot 2022-11-10 at 12 28 57](https://user-images.githubusercontent.com/2484268/201091789-7b8892a8-5144-4a08-8ab8-26a232547ce4.png)